downloads/swan-lake-release-notes/swan-lake-2201.2.0/RELEASE_NOTE.md

```diff
@@ -315,7 +315,7 @@ function getSum(int a, int b, int c) returns int {
 }
 ```
 
-#### Bug fixes
+#### Backward incompatible changes
 
 - A bug that resulted in allowing access to non-public error intersection types outside the module has been fixed.
```

Comment:
```suggestion
- Fixed a bug that resulted in allowing access to non-public error intersection types outside the module 
```
We have to start with the verb and remove the full-stop to preserve consistency with the other content.

Reply:
We've discussed this previously too, right? IMO, it isn't quite as clear. It may be misinterpreted as the bug fix resulted in allowing access to non-public error intersection types outside the module, which is not the case.